codenames/embedding_handler.py

```diff
@@ -13,10 +13,16 @@ class EmbeddingHandler:
     embedding_file : `str`
         Location of pickled embeddings
     """
-    def __init__(self, embedding_file: str):
+    def __init__(self, embedding_file: str) -> None:
```

Comment:
we should probably try to use the standard format embedding file format instead of a pickled file ?

Reply:
Done.